lib/ruby_lsp/requests/hover.rb

```diff
@@ -142,6 +151,35 @@ def generate_hover(name, location)
           contents: markdown_from_index_entries(name, entries),
         )
       end
+
+      sig { params(node: Prism::CallNode).void }
+      def generate_gem_hover(node)
+        first_argument = node.arguments&.arguments&.first
+        return unless first_argument.is_a?(Prism::StringNode)
+
+        spec = Gem::Specification.find_by_name(first_argument.content)
+        return unless spec
+
+        info = [spec.description, spec.summary, "This rubygem does not have a description or summary."].find do |text|
+          !text.nil? && !text.empty?
+        end
```

Comment:
I don't think we can use `#present?` here since it's an Active Support extension—not the prettiest thing in the world but I think it's roughly the same in terms of functionality based on [this guide](https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark).